src/BlueEnemy.cpp

```diff
@@ -15,6 +15,7 @@ void BlueEnemy::Draw(sf::RenderWindow& window)
 
 void BlueEnemy::Update()
 {
+    extern int WindowHeight; 
```

Comment:
No need for that - BlueEnemy includes Entity.h which has extern global width and height that it can use, just like other enemies

Reply:
sorry, I had deleted that but I revert previous commit so it appeared again :/
forgot to delete that line :c